terraform/modules/env-dashboards/dss-dashboard.tf

```diff
@@ -1,26 +1,133 @@
 locals {
   dss_dashboard = <<EOF
-{
-    "annotations": {
-        "list": [
-            {
-                "$hashKey": "object:267",
-                "builtIn": 1,
-                "datasource": "-- Grafana --",
-                "enable": true,
-                "hide": true,
-                "iconColor": "rgba(0, 211, 255, 1)",
-                "name": "Annotations & Alerts",
-                "type": "dashboard"
+        {
+            "annotations": {
+                "list": [
+                    {
+                        "$hashKey": "object:267",
+                        "builtIn": 1,
+                        "datasource": "-- Grafana --",
+                        "enable": true,
+                        "hide": true,
+                        "iconColor": "rgba(0, 211, 255, 1)",
+                        "name": "Annotations & Alerts",
+                        "type": "dashboard"
+                    }
+                ]
+            },
+            "editable": true,
+            "gnetId": null,
+            "graphTooltip": 0,
+            "iteration": 1543973320001,
+            "links": [],
+            "panels": [{
+            "aliasColors": {},
+            "bars": true,
+            "dashLength": 10,
+            "dashes": false,
+            "datasource": "${local.hca_logs_ttfb_datasource_name}",
+            "fill": 1,
+            "fillGradient": 0,
+            "gridPos": {
+                "h": 8,
+                "w": 24,
+                "x": 0,
+                "y": 8
+            },
+            "id": 22,
+            "legend": {
```

Comment:
please consider adding a legend and an average value within this legend for easier observation. the bucket panels have this if you need an example.

Reply:
I think it's cleaner as is.